Automattic/wp-calypso
# Merge edit focus plugins

Merges existing @Automattic/ajax plugins into FSE.

Next steps should be to add the feature flags (#33240) so we can get it ready to resubmit to the plugin directory.

### Testing
1. Check out branch and run `npx lerna run build --scope='@automattic/full-site-editing'` from wp-calypso's root
1. See the build succeeding and check that each plugin within `apps/full-site-editing/full-site-editing-plugin` has a `/dist` folder.
1. [Symlink to the plugin folder in a local test install](https://github.com/Automattic/wp-calypso/blob/master/apps/full-site-editing/README.md#local-development) and activate the plugin.
1. Create a new page and see if the template selector shows up. Select a template.
1. In the block selector, search for "posts list" and insert the posts list block.

Fixes #33239.
